themes/src/main/resources/theme/keycloak-preview/account/index.ftl

```diff
@@ -160,6 +178,10 @@
               <button id="landingSignOutButton" tabindex="0" style="display:none" onclick="keycloak.logout();" class="pf-c-button pf-m-primary" type="button">${msg("doSignOut")}</button>
             </div>
 
+            <div class="pf-l-toolbar__group" style="margin-left: 10px;">
+                <span id="loggedInUser"></span>
```

Comment:
I've noticed we use the same ID both at Welcome Screen and at React pages. The Welcome Screen is technically always present in the DOM, just hidden when a React page is loaded. I'm no HTML expert, so correct me if I'm wrong, but I think it's not valid to have duplicit IDs in HTML and it could theoretically cause some problems. One thing I can think of is testing – it could be a bit nondeterministic as we don't know which of the elements with this ID was selected.

At the Welcome Screen we usually use `landing` prefix for IDs to prevent this kind of issues.

Reply:
> At the Welcome Screen we usually use `landing` prefix for IDs to prevent this kind of issues.
I think that's a good idea.  We should make sure to do that from now on.